compiler/package.go

```diff
@@ -174,88 +159,28 @@ func Compile(importPath string, files []*ast.File, fileSet *token.FileSet, impor
 			return
 		}
 		// Some other unexpected panic, catch the stack trace and return as an error.
-		err = bailout(fmt.Errorf("unexpected compiler panic while building package %q: %v", importPath, e))
+		err = bailout(fmt.Errorf("unexpected compiler panic while building package %q: %v", importPath2, e))
```

Comment:
Could be considered a functional change, but while we're here, maybe...

```suggestion
		err = bailout(fmt.Errorf("unexpected compiler panic while building package %q: %w", importPath2, e))
```

Reply:
Had to undo this one because `e` is `interface{}`, not and `error` and `go vet` is upset about it.